tools/cockpit.spec

```diff
@@ -358,6 +358,7 @@ Provides: %{name}-subscriptions = %{version}-%{release}
 Requires: subscription-manager >= 1.13
 Provides: %{name}-networkmanager = %{version}-%{release}
 Requires: NetworkManager
+Requires: NetworkManager-team
```

Comment:
We actually get nice enough error messages when the plugin is not installed, so I think we are okay.